bgs/fedmgr.go

```diff
@@ -92,6 +92,8 @@ func (s *Slurper) RestartAll() error {
 	}
 
 	for _, pds := range all {
+		pds := pds
+		s.active[pds.Host] = &pds
```

Comment:
This line here was fixing another bug where we werent tracking active subscriptions started at bootup